algo/src/test/java/com/hartwig/actin/algo/evaluation/tumor/HasLymphNodeMetastasesTest.java

```diff
@@ -0,0 +1,32 @@
+package com.hartwig.actin.algo.evaluation.tumor;
+
+import com.hartwig.actin.algo.datamodel.Evaluation;
+import com.hartwig.actin.algo.datamodel.EvaluationResult;
+import org.junit.Test;
+
+import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;
+import static org.junit.Assert.assertTrue;
+
+public class HasLymphNodeMetastasesTest {
+
+    @Test
+    public void canEvaluate() {
```

Comment:
what about breaking this guy up into three tests for the different evaluation results? Would also result in an improved test name.

Reply:
I was following the convention from similar test classes but I agree with making this improvement.